cpp/include/cudf/detail/groupby/sort_helper.hpp

```diff
@@ -60,8 +60,9 @@ struct sort_groupby_helper {
    *                        optimizations to help skip re-sorting keys.
    */
   sort_groupby_helper(table_view const& keys,
-                      null_policy include_null_keys = null_policy::EXCLUDE,
-                      sorted keys_pre_sorted        = sorted::NO);
+                      null_policy include_null_keys                  = null_policy::EXCLUDE,
+                      sorted keys_pre_sorted                         = sorted::NO,
+                      std::vector<null_order> const& null_precedence = {});
```

Comment:
Please also remove these default values. We should stay as explicit as possible.